netkan/netkan/webhooks/github_inflate.py

```diff
@@ -49,6 +49,8 @@ def ids_from_commits(commits):
 
 
 def inflate(ids):
+    # Make sure our NetKAN repo is up to date
+    current_app.config['netkan_repo'].remotes.origin.pull('master', strategy_option='ours')
```

Comment:
In this instance, we probably want our remote to win. The indexer sets 'ours', because it's the source of the truth. The webhooks/schedulers are consumers of the data.